widgets/FTP-batchsource.json

```diff
@@ -18,6 +18,84 @@
           "widget-type": "textbox",
           "label": "Path",
           "name": "path"
+        },
+        {
+          "widget-type": "select",
+          "label": "Format",
+          "name": "format",
+          "widget-attributes": {
+            "default": "text",
+            "values": [
+              {
+                "label": "blob",
+                "value": "blob"
+              },
+              {
+                "label": "csv",
+                "value": "csv"
+              },
+              {
+                "label": "delimited",
+                "value": "delimited"
+              },
+              {
+                "label": "json",
+                "value": "json"
+              },
+              {
+                "label": "text",
+                "value": "text"
+              },
+              {
+                "label": "tsv",
+                "value": "tsv"
+              }
+            ]
+          }
+        },
+        {
+          "widget-type": "get-schema",
+          "widget-category": "plugin"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Delimiter",
+          "name": "delimiter",
+          "widget-attributes": {
+            "placeholder": "Delimiter if the format is 'delimited'"
+          }
+        },
+        {
+          "widget-type": "toggle",
+          "name": "enableQuotedValues",
```

Comment:
is delimiter and enableQuotedValues inherited in File source config?

Reply:
these aren't actually part of the file source config class, but the AbstractFileSource will pass anything that is set in the PluginConfig will down to the format plugins, which do use these values.

Reply:
I see, sounds good, thanks!